charts/aws-vpc-cni/README.md

```diff
@@ -5,6 +5,7 @@ This chart installs the AWS CNI Daemonset: https://github.com/aws/amazon-vpc-cni
 ## Prerequisites
 
 - Kubernetes 1.11+ running on AWS
```

Comment:
is k8s 1.11 still the oldest version supported?

Reply:
I am not sure on this one. I don't know if we have a way to determine the versions which are actively supported
@jayanthvn